Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -386,18 +386,19 @@ instance {ι : Type*} : Inhabited (HilbertBasis ι 𝕜 ℓ²(ι, 𝕜)) :=
 
 open Classical in
 /-- `b i` is the `i`th basis vector. -/
-instance instCoeFun : CoeFun (HilbertBasis ι 𝕜 E) fun _ => ι → E where
-  coe b i := b.repr.symm (lp.single 2 i (1 : 𝕜))
+@[coe] def toFun (b : HilbertBasis ι 𝕜 E) (i : ι) : E := b.repr.symm <| lp.single 2 i (1 : 𝕜)
```

Comment:
Just to not lose it from the other PR; this one should be adjusted to use FunLike, which needs the proofs in #21440 and probably the API in #21444

Reply:
I've reverted this. Changes were in commit 712d7f6c0c

Reply:
Sorry, which "other PR" does this refer to?

Reply:
"other PR" refers to comments in https://github.com/leanprover-community/mathlib4/pull/21395 on lines that were moved to this PR.